portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +0,0 @@
-// Copyright 2020 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
```

Comment:
Optional: Do you want to make use of this method (change the name if you do) to display a random favorite image of yours? It will get you some practice with using Javascript for later weeks. Or you can experiment with/think about using such function, but don't have to use it in your portfolio. Totally optional since you'll get some practice during Week 2 anyway.